docs/overview.md

```diff
@@ -1,19 +1,19 @@
 # Kubernetes User Documentation
 
-The Kubernetes API currently manages 3 main resources:
-* [pods](pods.md)
-* [replication controllers](replication-controller.md)
-* [services](services.md)
+Kubernetes is an open source orchestration system for containers (currently Docker containers). It schedules containers onto nodes in a compute cluster and actively manages these containers to ensure that the state of the cluster continually matches the user's intentions.
 
-In Kubernetes, rather than individual containers, _pods_ are the smallest deployable units that can be created, scheduled, and managed. Singleton pods can be created directly, and sets of pods may created, maintained, and scaled using replication controllers.  Services create load-balanced targets for sets of pods.
+In Kubernetes, containers run inside [pods](pods.md), which are an abstraction used to group one or more cooperating containers. Users can create a singleton pod, but the usual deployment pattern in Kubernetes is to define a pod and then to ask the system to create multiple identical replicas of that pod. For each pod the system finds a machine that is healthy and has sufficiently availability capacity, and starts up the corresponding container(s) there.
+The replicated set of pods might constitute an entire application, a micro-service, or one layer in a multi-tier application. 
 
-Kubernetes supports a unique [networking model](networking.md). Kubernetes encourages a flat address and does not dynamically allocate ports, instead allowing users to select whichever ports are convenient for them. To achieve this, it allocates an IP address for each pod and each service. Services provide stable addresses and [DNS names](dns.md) for clients to connect to, even as serving pods are replaced by new pods on new hosts.
+Kubernetes supports a unique [networking model](networking.md). Kubernetes encourages a flat address and does not dynamically allocate ports, instead allowing users to select whichever ports are convenient for them. To achieve this, it allocates an IP address for each pod.
```

Comment:
Fixed typo.
Didn't understand "and instead allows" comment -- if you meant the sentence should read "Kubernetes encourages a flat address and does not dynamically allocate ports, and instead allows users to select whichever ports are convenient for them" that sounds weird.  Or maybe you meant "Kubernetes encourages a flat address, does not dynamically allocate ports, and instead allows users to select whichever ports are convenient for them" ? Actually that sounds weird too. I think what's there is fine.